src/domain/collaboration/callout/dto/callout.dto.update.publisher.ts

```diff
@@ -0,0 +1,24 @@
+import { UUID_NAMEID_EMAIL } from '@domain/common/scalars';
+import { Field, InputType } from '@nestjs/graphql';
+
+@InputType()
+export class UpdateCalloutPublisherInput {
+  @Field(() => String, {
+    nullable: false,
+    description:
+      'The identifier for the Callout whose publisher is to be updated.',
+  })
+  calloutID!: string;
+
+  @Field(() => UUID_NAMEID_EMAIL, {
+    nullable: false,
+    description: 'The identifier of the publisher of the Callout.',
+  })
+  publisherID!: string;
+
+  @Field(() => Number, {
+    nullable: true,
+    description: 'The timestamp to set for the publishing of the Callout.',
+  })
+  publishDate!: number;
+}
```

Comment:
This dto is named `UpdateCalloutPublisherInput` but it is changing both the publisher and publishDate. Consider changing the name.

Reply:
Ok, will rename to UpdateCalloutPublishInformationInput